code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -25,6 +25,7 @@
 	melee_damage_upper = 3
 	ai_controller = /datum/ai_controller/monkey
 	faction = list("neutral", "monkey")
+	possible_a_intents = list(INTENT_HELP, INTENT_DISARM, INTENT_GRAB, INTENT_HARM)
```

Comment:
note that monkeys have a special disarm attack that's basically a curator whip effect (with a 5% chance of a knockdown (or a hardstun if the target is already knocked down when you roll that chance)), not a shove attack like the ones humans use. I don't think it'll be much of an issue (especially since the aforementioned curator whip exists, but nobody really uses it much), but I figured that I should at least make you aware of it (if you weren't already).

Reply:
It seemed like a regular shove to me when I tested it

Reply:
I had to add what happens when they disarm, before nothing was happening